src/pythermalcomfort/models.py

```diff
@@ -3432,6 +3434,7 @@ def __init__(
         weight_loss_by_evap_and_res: weight loss by the evaporation and respiration of the whole body [g/sec]
         OPTIONAL PARAMETERS : the paramters listed below are returned if ex_output = "all"
         Q_bmr_core: core heat production by basal metabolism (each body part) [W]
```

Comment:
This is referred to the previous comment, as you can see all the Q are capitalised, it would be nice to have them as q_....

Reply:
The same as above.